07-routing-gossip.md

```diff
@@ -0,0 +1,113 @@
+# BOLT #7: P2P Node and Channel Discovery
+
+This specification describes a simple initial node and channel discovery mechanism that does not rely on a third-party to disseminate the information.
+
+Node and channel discovery serve two different purposes:
+
+ - Channel discovery allows the creation and maintenance of a local view of the network's topology such that the node can discover routes to desired destination.
+ - Node discovery allows nodes to broadcast their ID, host and port, such that other nodes can open connections and establish payment channels.
+ 
+Peers in the network exchange `route_announcement` messages that contain information about a node and about its outgoing channels.
+The node creating an announcement is referred to as the _announcing node_.
+Announcements MUST have at least one associated channel whose existence can be proven by inspection of the blockchain, i.e., the anchor transaction creating the channel MUST have been confirmed.
```

Comment:
Btw I just realized that 0-conf anchors are not possible with ids based on block height 😁